components/SetUpStaking/requests.js

```diff
@@ -0,0 +1,108 @@
+
+import { useReadContract } from 'wagmi';
+import { base } from 'wagmi/chains';
+import { readContract, writeContract, waitForTransactionReceipt } from '@wagmi/core';
+import {
+  CONTRIBUTE_MANAGER_ADDRESS_BASE,
+  CONTRIBUTE_STAKING_INSTANCE_ADDRESS_BASE,
+  STAKING_TOKEN_ABI,
+  OLAS_ADDRESS_BASE,
+  GENERIC_ERC20_ABI,
+  CONTRIBUTE_MANAGER_ABI
+} from 'common-util/AbiAndAddresses';
+import { wagmiConfig } from 'common-util/Login/config';
+
+export const useTotalBond = () => {
+  const { data, isLoading } = useReadContract({
+    address: CONTRIBUTE_STAKING_INSTANCE_ADDRESS_BASE,
+    abi: STAKING_TOKEN_ABI,
+    chainId: base.id,
+    functionName: 'minStakingDeposit',
+  });
+
+  // Calculate the total bond required for the service deployment
+  // by the formula: (1 + NUM_AGENT_INSTANCES) * minStakingDeposit
+  // where NUM_AGENT_INSTANCES always equals to 1 for this contract
```

Comment:
Nice :)

Reply:
Can we define that `2` constant somewhere as a variable and use that in the value field as well?